src/commands/interviews/interviewers.ts

```diff
@@ -0,0 +1,61 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import _ from 'lodash';
+
+import { BaseCommand } from '../../utils/commands';
+import { availableDomains, getAvailableDomainsString, getInterviewers, Interviewer } from '../../components/interview';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import { parseDomainArg, validateDomainArg } from './utils';
+
+const RESULTS_PER_PAGE = 6;
+
+class InterviewersCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'interviewers',
+      aliases: ['interviewer', 'interviewer-list', 'interviewers-list'],
+      group: 'interviews',
+      memberName: 'interviewers',
+      args: [
+        {
+          key: 'domain',
+          prompt: `enter one of ${getAvailableDomainsString()}.`,
+          type: 'string',
+          default: '',
+          validate: validateDomainArg,
+          parse: parseDomainArg
+        }
+      ],
+      description: 'Shows you a list of available interviewers.',
+      examples: ['.interviewer signup calendly.com/codey/mock-interview']
```

Comment:
Should be examples of interviewer list, not interviewer signup.

Reply:
good catch :)